pkg/kubefedctl/enable/validation.go

```diff
@@ -114,28 +114,16 @@ func federatedTypeValidationSchema(templateSchema map[string]v1beta1.JSONSchemaP
 											"path": {
 												Type: "string",
 											},
+											// Supporting the override of an arbitrary field
+											// precludes up-front validation.  Errors in
+											// the definition of override values will need to
+											// be caught during propagation.
 											"value": {
-												// Supporting the override of an arbitrary field
-												// precludes up-front validation.  Errors in
-												// the definition of override values will need to
-												// be caught during propagation.
-												AnyOf: []v1beta1.JSONSchemaProps{
-													{
-														Type: "string",
-													},
-													{
-														Type: "integer",
-													},
-													{
-														Type: "boolean",
-													},
-													{
-														Type: "object",
-													},
-													{
-														Type: "array",
-													},
-												},
+												// TODO(marun) Set x-kubernetes-preserve-unknown-fields to ensure
+												// unknown fields won't be stripped once `preserveUnknownFields: true`
```

Comment:
My confusion is in the TODO that states:
> unknown fields won't be stripped once `preserveUnknownFields: true` becomes the default for CRDs in 1.16. 

It's my understanding that pruning is the default in `apiextensions.k8s.io/v1` (1.16) which means `preserveUnknownFields` is `false` instead of `true`. Am I missing something?

Reply:
Yes, the todo is incorrect.  unknown fields will be stripped once false becomes the default.